pkg/cli/types.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	colorable "github.com/mattn/go-colorable"
+)
+
+type IO struct {
+	In  io.Reader
+	Out io.Writer
+	Err io.Writer
+}
+
+func DefaultIO() *IO {
+	io := &IO{
+		In:  os.Stdin,
+		Out: os.Stdout,
+		Err: os.Stderr,
+	}
+	if runtime.GOOS == "windows" {
+		io.Out = colorable.NewColorableStdout()
+		io.Err = colorable.NewColorableStderr()
+	}
+	return io
+}
+
+type RootDir string
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type RootDir should have comment or be unexported

Reply:
fixed in 3041c36